docs/UserGuide.md

```diff
@@ -228,96 +410,200 @@ The following configuration options currently exist:
+  - `MODULE_CODE`: The module code of the module associated with this task. Must be a single word containing only alphanumeric characters and underscore `_`. Furthermore, a module with this module code must currently exist.
+  <br><br>
+  ##### Example 1:
+  ```
+  >  mark c 1
+  ```
+  ##### Expected Output:
+  ```
+  ____________________________________________________________
+  Nice! I have marked this task as completed!
+  (X) Reply emails []
+  ____________________________________________________________
+  ```
+  ##### Example 2:
+  ```
+  >  mark u 1 -m CS2113T
+  ```
+  ##### Expected Output:
+  ```
+  ____________________________________________________________
+  Ok! I have marked this task for you as uncompleted!
+  ( ) CS2113T Tutorial 2 []
+  ____________________________________________________________
+  ```
 <br>
 
 ### 4.7. Managing custom tags: `tag`
 
-Allows you to add or delete a tag from the [specified task](#32-specifying-tasks).
-
-> ⚠ <span style="color:#ffa500">**IMPORTANT:**</span>
->
-> The tag name must be a single word; it cannot contain whitespace.
+- Allows you to add or delete a tag from the [specified task](#32-specifying-tasks).
 
-Format: `tag (add | del) TASK_NUMBER [-m MODULE_CODE] TAG_NAME`
-- `TASK_NUMBER`: The number of the task to be deleted. Must be a positive integer.
-- `MODULE_CODE`: The module code of the module associated with this task. Must be a single word containing only alphanumeric characters and underscore `_`. Furthermore, a module with this module code must currently exist.
-- `TAG_NAME`: The name of the tag to be added or deleted. Must be a single word containing only alphanumeric characters and underscore `_`. 
+  > ⚠ <span style="color:#ffa500">**IMPORTANT:**</span>
+  >
+  > The tag name must be a single word; it cannot contain whitespace.
 
-Example: `tag add 1 -m CS2113T project`
+  #### Format: `tag (add | del) TASK_NUMBER [-m MODULE_CODE] TAG_NAME`
+  - `TASK_NUMBER`: The number of the task to be deleted. Must be a positive integer.
+  - `MODULE_CODE`: The module code of the module associated with this task. Must be a single word containing only alphanumeric characters and underscore `_`. Furthermore, a module with this module code must currently exist.
+  - `TAG_NAME`: The name of the tag to be added or deleted. Must be a single word containing only alphanumeric characters and underscore `_`. 
+  ##### Example:
+  ```
+  >  tag add 1 -m CS2113T project
+  ```
+  ##### Expected Output:
+  ```
+  ____________________________________________________________
+  Tag "project" added:
+  ( ) CS2113T Tutorial 2 [project].
+  ____________________________________________________________
+  ```
 
 <br>
 
 ### 4.8. Listing all tasks: `list`
 
-Shows you your tasks, grouped by module code. General tasks are displayed separately.
+- Shows you your tasks, grouped by module code. General tasks are displayed separately.
```

Comment:
Maybe consider removing the bullet point for the sections with only one subsection

Reply:
i think we will keep it like this as of now and see if its an issue in the future